types/src/block/output/alias_id.rs

```diff
@@ -19,6 +22,11 @@ impl AliasId {
     pub fn or_from_output_id(self, output_id: &OutputId) -> Self {
         if self.is_null() { Self::from(output_id) } else { self }
     }
+
+    /// Returns the bech32 encoding of the alias ID.
+    pub fn to_bech32(&self, bech32_hrp: &str) -> String {
```

Comment:
maybe we should make this a trait?

Reply:
https://github.com/iotaledger/iota-sdk/issues/18